themes/digital.gov/layouts/shortcodes/ring.html

```diff
@@ -0,0 +1,29 @@
+<!-- prettier-ignore-start -->
+{{/* Displays a blue ring around a piece of content in order to highlight it
+
+Can accept nested text based shortcodes like note and box.
+Img shortcodes will not render, do not use within ring shortcode
+
+Shortcode usage:
+{{< ring title="Case in point">}}
+This is a ring shortcode
+{{< /ring >}}
+
+With nested shortcodes, text based only:
+{{< ring title="Nested text onyl shortcodes" >}}
+{{< note "plain" >}}
+**Note Goes Here** More note content goes into here
+{{< /note >}}
+{{< /ring >}}
+*/}}
+<!-- prettier-ignore-end -->
+
+{{- $title := (.Get "title") -}}
+
+
+<div class="dg-ring">
+  <h1 class="dg-ring__title">
+    {{- $title -}}
+  </h1>
+  {{ .Inner | markdownify }}
+</div>
```

Comment:
Using a generic element like `div` will introduce a11y issues with heading order.

At the very least we should change this to an `<article>` or `<section>`.

```suggestion
<article class="dg-ring">
  <h1 class="dg-ring__title">
    {{- $title -}}
  </h1>
  {{ .Inner | markdownify }}
</article>
```

---

Going forward we should generate a random ID to tie the header to the container for a better screen reader experience.

```diff
+<article class="dg-ring" aria-labelledby="{{ randomHeaderID }}">
+  <h1 id="#{{ randomHeaderID }} class="dg-ring__title">
    {{- $title -}}
  </h1>
  {{ .Inner | markdownify }}
</article>
```